HW02/vorobyov_egor/spec/my_array_spec.rb

```diff
@@ -0,0 +1,56 @@
+require('../my_array')
+
+RSpec.describe MyArray do
+  my_arr = MyArray.new([1, 2, 3, 4, 5])
+  empty_arr = []
+
+  describe '#select' do
+    it 'returns an array of trueth values' do
+      trueth_values = [2, 4]
+      expect(my_arr.select { |el| el.even? }).to eq trueth_values
+    end
+
+    it "returns NoMethodError when block wasn't given" do
+      expect { my_arr.select }.to raise_error(NoMethodError)
+    end
```

Comment:
```suggestion
    context 'when block wasn't given' do
      it "returns NoMethodError" do
        expect { my_arr.select }.to raise_error(NoMethodError)
      end
    end
```

similarly for others